MinGW has all these headers, but we want the #elif defined WIN32 block later

--- origsrc/pcre2-10.23/src/pcre2grep.c	2017-02-12 07:29:54.000000000 -0600
+++ src/pcre2-10.23/src/pcre2grep.c	2017-03-22 00:54:10.263533500 -0500
@@ -628,7 +628,7 @@ z/OS, and "no support". */
 
 /************* Directory scanning Unix-style and z/OS ***********/
 
-#if (defined HAVE_SYS_STAT_H && defined HAVE_DIRENT_H && defined HAVE_SYS_TYPES_H) || defined NATIVE_ZOS
+#if (defined HAVE_SYS_STAT_H && defined HAVE_DIRENT_H && defined HAVE_SYS_TYPES_H && !defined WIN32) || defined NATIVE_ZOS
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <dirent.h>
